.circleci/config.yml

```diff
@@ -17,13 +17,17 @@ jobs:
           name: "Install yarn at specific version"
           command:
             sudo npm install --global yarn@1.13.0
-
       - run:
           name: "Show yarn and node versions"
           command: |
             node --version
             yarn --version
       - checkout
+      - restore_cache:
```

Comment:
this restores node_modules/ if it exists in the cache, the hash key for the cache is the `yarn.lock` file